### PR TITLE
Readapt no longer breaks hivemind.

### PR DESCRIPTION
**Pull request by ghost**
Removing this line seems to have no effect on admins dechangelinging someone.
I tested it 1 player at first to see if I could still broadcast messages on hivemind after a changeling/dechangeling and changeling readapt. they couldn't in the first case and could in the second one.
I tested 2 lings, one I controlled and the other one I forced say'd something to ling chat to check if I could hear it. I could.
I delinged the forcesay ling and tested it again and I couldn't hear them in ling chat but I could hear them in local chat.

### CODE REVIEW

**Comment by Carbonhell** [comment]
Wait what do you mean with your last statement?Is there any bug that this PR creates or what?

**Comment by Carbonhell** [comment]
Apparently whoever made lings or touched its code though that lings should be able to be restricted from using :g. I think this is pretty retarded nor it's used anywhere in the code pratically. I'd suggest removing all the instances of changeling_speak var.

**Comment by blobbernaut** [comment]
i hate changelings a lot but i hate bugs too and this one was quite annoying

**Comment by myontasm** [comment]
Why the fuck would this be a thing in the first place? :+1:

**Comment by KayeArray** [comment]
I thought this was feature, now yousa telling me this shit was a bug all along? well fuck

**Comment by ghost** [comment]
Okay, removed all instances and fixed some other bugs too. Can this get a chode check pleaseirino?

File: code/game/gamemodes/changeling/powers/hivemind.dm [hunk 1]
```diff
@@ -1,32 +1,11 @@
...
-	helptext = "We will be able to talk with other changelings with :g. Exchanged DNA does not count towards absorb objectives."
-	evopoints_cost = 0
-	chemical_cost = -1
-
-/obj/effect/proc_holder/changeling/hivemind_comms/on_purchase(var/mob/user)
-	..()
-	var/datum/changeling/changeling=user.mind.changeling
-	changeling.changeling_speak = 1
-	user << "<i><font color=#800080>Use say \":g message\" to communicate with the other changelings.</font></i>"
-	var/obj/effect/proc_holder/changeling/hivemind_upload/S1 = new
```
Comment (Carbonhell) [inline code/game/gamemodes/changeling/powers/hivemind.dm:14]: you sure readapt doesn't delete those 2 spells?

**Comment by Carbonhell** [comment]
Code is ok

**Pull request merged**